astor/code_gen.py

```diff
@@ -226,6 +226,32 @@ def body_or_else(self, node):
         self.body(node.body)
         self.else_body(node.orelse)
 
+    def docstring(self, node, indent=1):
+        """
+        Write the function/module/class docstring attribute
```

Comment:
Style nit: Please finish the sentence with a period.

Reply:
Period added.